web/templates/confirm_dialog/confirm_delete_detached_attachments.hbs

```diff
@@ -0,0 +1,11 @@
+<p>
+    {{t "Would you like to delete the following uploaded files? If you do so, they will no longer be accessible from the edit history for this message."}}
```

Comment:
@alya this copy may need an if/else clause for whether message edit history is enabled in the organization, or a help link or something.

Reply:
"The following uploaded files are now only accessible via this message's <a>message edit history</a>. Would you like to delete them?", with the link being a link to the /help/ article for message edit history.

Reply:
And what about `Manage uploaded files` link? Should we show that only on error?

Reply:
> `The following uploaded files are now only accessible via this message's <a>edit history</a>. Would you like to delete them?`, with the link being a link to the /help/ article for message edit history.

Mm, yeah, I like that direction. Maybe something like this?

- If edit history enabled: `The following uploaded files are now only accessible via this message's <a>edit history</a>. Would you like to delete them?`
- If edit history disabled: `The following uploaded files are now only accessible via this message's <a>edit history</a>, which is currently disabled in this organization. Would you like to delete them?`

I'm hoping to hint that the files would become accessible if edit history were to be enabled in the future.

Reply:
> And what about Manage uploaded files link? Should we show that only on error?

I think it's fine to always show this.